script/Arcueid/Day 14/14_00_ARC14_1.txt

```diff
@@ -801,7 +800,7 @@ Neither one of us says another word.
 [sha:663fc05fb8daacf24bbc5a475e39ebc65e44dc1e]{
 -- Page 52, Offset 36901.
 -- 本当にどうかしてしまいそうな沈黙のあと。
-After a silence that felt like an eternity―――
+After an almost maddening silence―――
```

Comment:
revert

Reply:
Really? It feels like we're deviating too much here.

Reply:
iirc this was an intentional localization because "almost maddening silence" or "nearly maddening silence" is quite awkward in English and what it needs is "silence that feels hyperbolic-ly long/painful".

Reply:
Yeah I'd localized this one intentionally for the reasons Ben described